content/millennium/r4/individuals/person.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Person | R4 API
+---
+
+# Person
+
+* TOC
+{:toc}
+
+## Overview
+
+The Person resource identifies an individual outside of a specific health care context providing a mechanism to link person resources across different facilities or organizations.
+
+The following fields are returned if valued:
+
+* [Person id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Identifier (Cerner alias Federated Person Principal)](http://hl7.org/fhir/R4/person-definitions.html#Person.identifier){:target="_blank"}
+* [Name](http://hl7.org/fhir/R4/person-definitions.html#Person.name){:target="_blank"}
+* [Telecom](http://hl7.org/fhir/R4/person-definitions.html#Person.telecom){:target="_blank"}
+* [Gender](http://hl7.org/fhir/R4/person-definitions.html#Person.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/person-definitions.html#Person.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/person-definitions.html#Person.address){:target="_blank"}
+* [Active](http://hl7.org/fhir/R4/person-definitions.html#Person.active){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:person, :r4) %>
+
+## Search
+
+Search for Persons that meet supplied query parameters:
+
+    GET /Person?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Parameters
+
+ Name                 | Required?                                    | Type       | Description
+----------------------|----------------------------------------------|------------|--------------------------------------------------------------------------
+ `_id`                | No, if the identifier is populated           | [`token`]  | The logical resource id associated with the resource.
+ `identifier`         | Yes, or `_id`                                | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```

Comment:
Yes, or `_id` -> This or `_id`

Reply:
Changed in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc